docs/source/en/api/pipelines/ldm3d_diffusion.mdx

```diff
@@ -0,0 +1,60 @@
+<!--Copyright 2023 The HuggingFace Team. All rights reserved.
```

Comment:
Nice doc! Could we maybe also add a line to the public pipeline table here: https://github.com/huggingface/diffusers/blob/8caa530069afab4a0462218da96231eee19dcc24/docs/source/en/index.mdx?plain=1#L66

Reply:
Done!